src/librustc_passes/ast_validation.rs

```diff
@@ -91,19 +91,6 @@ impl<'a> AstValidator<'a> {
         }
     }
 
-    fn no_questions_in_bounds(&self, bounds: &TyParamBounds, where_: &str, is_trait: bool) {
```

Comment:
You've removed all the calls to this method. I infer that it because you want to assume that traits automatically start off as implementing `Move`, and thus you need to be able to put `: ?Move` as bound on trait definitions in order to override that initial default.

But shouldn't we replace the old function with one that is specialized to `?Sized`? Or is there some reason why we want to stop erroring in response to something like `trait X : ?Sized { fn x(&self); }` ?

Reply:
oh never mind, I see that you did add such a check in `fn conv_object_ty_poly_trait_ref`